FastSimulation/ForwardDetectors/plugins/ProtonTaggerFilter.cc

```diff
@@ -290,7 +279,7 @@ bool ProtonTaggerFilter::filter(edm::Event& iEvent, const edm::EventSetup& es) {
 
       acc420or220b1 = acc420b1 + acc220b1 - acc420and220b1;
 
```

Comment:
This is only used for the commented out log message below: better to also comment it out, not only for the tiny performance improvement, but also to avoid mistakes when using those variables with quite similar names
```suggestion
      //acc420or220b1 = acc420b1 + acc220b1 - acc420and220b1;
```